types/file.go

```diff
@@ -10,6 +10,7 @@ type DiskStat struct {
 	DiskID           string
 	Path             string
 	Type             string
+	Driver           string
```

Comment:
Could you point to where this will be assigned with a value?

Reply:
Yeah, let me move it into another PR.

Reply:
> Could you point to where this will be assigned with a value?

longhorn-manager's disk_utils.go